src/main/java/com/puppycrawl/tools/checkstyle/gui/MainFrameModel.java

```diff
@@ -63,7 +63,7 @@
          * Provides description.
          * @param descr description
          */
-        ParseMode(String descr) {
+        /* default */ ParseMode(String descr) {
```

Comment:
here

Reply:
done

Reply:
Now it does  not compile. The PMD rule is broken on enum constructors

Reply:
@pbludov Eclipse is telling me the enum constructor has to be `private`, but we marks this as redundant through our checks
````
[ERROR] M:\checkstyleWorkspace\checkstyle\src\main\java\com\puppycrawl\tools\checkstyle\gui\MainFrameModel.java:66:9: Redundant 'private' modifier. [RedundantModifier]
````
I think this has to be `default` not matter what.

Reply:
@romani See above.

Reply:
done